### DISCUSSION TITLE
What is the recommended file size limit compared to available RAM?

### DISCUSSION THREAD

**Discussion by dankovas** [atoti/atoti#492] [Q&A] [answered]
Hi, does anybody knows what the maximum file size i can load in Jyputer notebook compared to RAM on my machine?

**Comment by tibdex** [comment, marked as answer]
The recommendation for the size of the data loaded in an atoti session is to stay under 33% of the RAM allocated to the session. You can change how much of your machine's RAM your session can take using [Java options](https://docs.atoti.io/0.6.4/lib/atoti.config.session_config.html#atoti.config.session_config.SessionConfig.java_options). You can also read [this](https://docs.atoti.io/0.6.4/deployment/deployment_setup.html#java-options).

**Reply by dankovas** [reply]
Great, thanks a lot! I will play with heap settings. Does the performance deteriorate if data size surpasses 33%?

**Reply by tibdex** [reply]
> Does the performance deteriorate if data size surpasses 33%?

It can yes as less memory will be available for queries so their execution will start taking more time.
